core/src/main/java/org/infinispan/container/versioning/SimpleClusteredVersion.java

```diff
@@ -24,14 +25,13 @@
    /**
     * The cache topology id in which it was first created.
     */
-   @ProtoField(number = 1, defaultValue = "0")
-   int topologyId;
+   @ProtoField(number = 1, defaultValue = "-1")
+   final int topologyId;
 
-   @ProtoField(number = 2, defaultValue = "0")
-   long version;
-
-   SimpleClusteredVersion() {}
+   @ProtoField(number = 2, defaultValue = "-1")
+   final long version;
```

Comment:
The default for version is -1 here, but in its sibling NumericVersion it is 0. I think it's better to make it -1 in both.